lib/PHPExif/Mapper/Exiftool.php

```diff
@@ -401,7 +401,7 @@ protected function extractGPSCoordinates(string $coordinates) : float|false
                 return false;
             }
 
-            return floatval($matches[1]) + (floatval($matches[2]) / 60) + (floatval($matches[3]) / 3600);
+            return round(floatval($matches[1]) + (floatval($matches[2]) / 60) + (floatval($matches[3]) / 3600), 12);
```

Comment:
Could we add a comment, why we are using 12 decimal digits? Why not 10 or 14?

Maybe should also add a constant `public const ROUNDING_PRECISION = 12` to the common interface `MapperInterface` and then use this constant here and in `ImageMagick`. See below.